tema-html-1/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Programul meu</h1>
+    <label> <i>Mai 2023</i> <progress value="25" max="31">25</progress></label>
+    <nav>
+        <a href="bio.html">Bio</a>
+        <a href="blog.html">Blog</a>
+    </nav> <hr>
+    <table border>
+        <thead>
+            <th style="width: 50px;"></th>
+            <th>Luni</th>
+            <th>Marti</th>
+            <th>Miercuri</th>
+            <th>Joi</th>
+            <th>Vineri</th>
+            <th>Sambata</th>
+            <th>Duminica</th>
+        </thead>
+        <tbody>
+            <tr>
+                <td><h1>6-8</h1></td>
+                <td>Rutina de dimineata</td>
+                <td>Rutina de dimineata</td>
+                <td>Rutina de dimineata</td>
+                <td>Rutina de dimineata</td>
+                <td>Rutina de dimineata</td>
+                <td>Sigur dorm la ora asta</td>
+                <td>Aici la fel</td>
+            </tr>
+            <tr>
+                <td><h2>8-16</h2></td>
+                <td>Serviciu</td>
+                <td>Serviciu</td>
+                <td>Serviciu</td>
+                <td>Serviciu</td>
+                <td>Serviciu</td>
+                <td>Dau o tura cu masina</td>
+                <td></td>
+            </tr>
+            <tr>
+                <td><h3>16-18</h3></td>
+                <td>Ma plimb prin parc</td>
+                <td>Spal masina</td>
+                <td>Ies la o inghetata</td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+            </tr>
+            <tr>
+                <td><h4>18-21</h4></td>
+                <td><img src="/tema-html-1/content/html.png" alt="HTML" width="40px" height="40px"></td>
+                <td>Fac cumparaturi</td>
+                <td>Pregatesc temele pentru curs</td>
+                <td><img src="/tema-html-1/content/html.png" alt="HTML" width="40px" height="40px"></td>
+                <td></td>
+                <td>Repet ce am invatat la curs</td>
+                <td>Repet ce am invatat la curs</td>
+            </tr>
+            <tr>
+                <td><h5>21-23</h5></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+            </tr>
+        </tbody>
+    </table>
+    <hr>
+    <footer>&copy; <i>Condurachi Razvan, Mai, 2023</i> </footer>
+
+    
+    
+</body>
+</html>
```

Comment:
Cerinta pentru pagina de index ar fi  in principiu sa reproduci 1 la 1 cu ce este in poza 😄 dar e close enough